src/main/java/crawlercommons/robots/BaseRobotRules.java

```diff
@@ -124,14 +112,17 @@ public String toString() {
         } else {
             sb.append(" - crawl delay: ").append(delay).append('\n');
         }
+
         int nSitemaps = getSitemaps().size();
         if (nSitemaps == 0) {
             sb.append(" - no sitemap URLs\n");
         } else {
             sb.append(" - number of sitemap URLs: ").append(nSitemaps).append('\n');
-            sb.append("   sitemaps[0]: ").append(getSitemaps().get(0)).append('\n');
+            if (nSitemaps <= 10) {
+                sb.append(String.join("\n", getSitemaps())).append("\n\n");
+            }
```

Comment:
Why not append up to 10 sitemaps, versus appending nothing if there or more than 10?

Reply:
I followed the same logic which I found in the [SimpleRobotRules](https://github.com/crawler-commons/crawler-commons/blob/e55bf1f6971bddd142316eb4e009c728541495f2/src/main/java/crawlercommons/robots/SimpleRobotRules.java#L369)

But if you think it makes more sense, I can change both files to add the first 10 sitemaps/robotRules